webapi/src/test/scala/org/knora/webapi/messages/admin/responder/valueObjects/ProjectsValueObjectsADMSpec.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright © 2021 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.messages.admin.responder.valueObjects
+
+import org.knora.webapi.UnitSpec
+import org.knora.webapi.exceptions.BadRequestException
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectsErrorMessagesADM._
+import org.knora.webapi.messages.store.triplestoremessages.StringLiteralV2
+import zio.prelude.Validation
+
+/**
+ * This spec is used to test the [[ProjectsValueObjectsADM]] value objects creation.
+ */
+class ProjectsValueObjectsADMSpec extends UnitSpec(ValueObjectsADMSpec.config) {
+  "ProjectIRI value object" when {
+//    TODO: check string formatter project iri validation because passing just "http://rdfh.ch/projects/@@@@@@" works
+    val validProjectIri = "http://rdfh.ch/projects/0001"
+
+    "created using empty value" should {
+      "throw BadRequestException" in {
+        ProjectIRI.make("") should equal(Validation.fail(BadRequestException(PROJECT_IRI_MISSING_ERROR)))
+      }
+    }
+    "created using invalid value" should {
+      "throw BadRequestException" in {
+        ProjectIRI.make("not a project IRI") should equal(
+          Validation.fail(BadRequestException(PROJECT_IRI_INVALID_ERROR))
+        )
+      }
+    }
+    "created using valid value" should {
+      "not throw BadRequestExceptions" in {
+        ProjectIRI.make(validProjectIri) should not equal Validation.fail(
+          BadRequestException(PROJECT_IRI_INVALID_ERROR)
+        )
+      }
```

Comment:
Does it need this test? If the next test ("return value passed to value object") passes, it's enough I think.